Modules/Core/Mesh/test/itkCellInterfaceTest.cxx

```diff
@@ -44,7 +44,7 @@ using CellAutoPointer = CellType::CellAutoPointer;
 
 template <typename TCell>
 int
-TestCellInterface(std::string name, TCell * aCell)
+TestCellInterface(const std::string & name, TCell * aCell)
```

Comment:
Of course, the `name` parameter type of this `TestCellInterface` could also be `string_view`.